spec/stat_tracker_spec.rb

```diff
@@ -107,4 +107,15 @@
       expect(stat_tracker.count_of_teams).to be_a(Integer)
     end
   end
+
+    it "#highest_scoring_visitor" do 
+    # require 'pry';binding.pry
+      expect(stat_tracker.highest_scoring_visitor).to be_a(String)
+      expect(stat_tracker.highest_scoring_visitor).to eq( "FC Dallas")
+    end
+
+    it "#lowest_scoring_visitor" do 
+      expect(stat_tracker.lowest_scoring_visitor).to be_a(String)
+      expect(stat_tracker.lowest_scoring_visitor).to eq( "San Jose Earthquakes")
```

Comment:
Clear tests, I like the addition of the expected output once we're able to get the desired team_name

Reply:
for sure. Not sure if we're supposed to be using the spec harness tests to gauge our accuracy, but might as well use the info while we have it?